pages/rops/update/schema/index.js

```diff
@@ -140,7 +145,11 @@ module.exports = req => {
       format: toBoolean,
       automapReveals: true,
       options: [
-        false,
+        {
+          value: false,
+          disabled: disableProcRadio('endangered'),
+          ...(disableProcRadio('endangered') && {warning: content.fields.disabledWarning})
```

Comment:
Further thoughts this could then be 
```suggestion
          reveal: disableProcRadio('endangered') ? <Warning><Snippet>fields.disabledWarning</Snippet></Warning> : undefined
```
And we don't need the very specific addition of a warning reveal to the generic react-component

Reply:
Yes I am raising another PR for react-component to make it generic.. as suggested

Reply:
I'd say just undo the previous changes to react-components and use the existing reveal prop?

Reply:
https://github.com/UKHomeOffice/react-components/pull/102